### PR TITLE
Added method for ACH verification

### PR DESCRIPTION
**Pull request by abdyfranco**

### CODE REVIEW

File: stripe_payments.php [hunk 1]
```diff
@@ -1223,63 +1239,56 @@ public function storeAch(array $account_info, array $contact, $client_reference_
...
+        $account = $this->handleApiRequest(
+            ['Stripe\Customer', 'retrieveSource'],
+            [$client_reference_id, $account_reference_id],
+            $this->base_url . 'customers - retrieveSource'
+        );
 
-            if ($this->Input->errors()) {
-                return false;
-            }
+        if (!empty($account['error'])) {
```
Comment (JReissmueller) [inline stripe_payments.php:1270]: I think $account is an object, but also I don't think we want to ignore errors in fetching the account.

File: stripe_payments.php [hunk 2]
```diff
@@ -1120,6 +1112,30 @@ public function buildAchForm($account_info = null)
...
+     */
+    public function buildAchVerificationForm(array $vars = [])
+    {
+        return null;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function requiresAchVerification()
```
Comment (JReissmueller) [inline stripe_payments.php:1134]: If you agree with my comment on the core PR then this should be removed.

**Review by JReissmueller — APPROVED** [review]

**Review by JReissmueller — CHANGES_REQUESTED** [review]
Meant to request changes

File: stripe_payments.php (hunk 1 above)
Comment (abdyfranco) [reply stripe_payments.php]: $account is an object only on a successful response, when an error occurs, an array is returned.

**Review by JReissmueller — APPROVED** [review]

**Pull request merged**